lib/error_tracker.ex

```diff
@@ -205,6 +204,24 @@ defmodule ErrorTracker do
     Process.get(:error_tracker_context, %{})
   end
 
+  @spec set_context(String.t()) :: list(String.t())
```

Comment:
Small typo in the function name:

```suggestion
  @spec add_breadcrumb(String.t()) :: list(String.t())
```